Tools/cases_generator/py_metadata_generator.py

```diff
@@ -1,6 +1,6 @@
-"""Generate uop metedata.
+"""Generate opcode metadata.
```

Comment:
Maybe add "for Python" to distinguish it from the previous file?

Reply:
Good point. Done